src/calculateRentalCost.js

```diff
@@ -1,10 +1,27 @@
 /**
- * @param {number} days
- *
- * @return {number}
- */
-function calculateRentalCost(days) {
-  // write code here
+* @param {number} days
+*
+* @return {number}
+*/
```

Comment:
fix issues with comments and try to rerun npm test, there shouldn't be any errors